app/components/grid.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react'
+import clsx from 'clsx'
+import type {ReactNode} from 'react'
+
+interface GridProps {
+  children: ReactNode
+  overflow?: boolean
+  className?: string
+}
+
+function Grid({children, className}: GridProps) {
```

Comment:
The base layout component. So far, everything besides the navbar is wrapped in it. The column counts and gap sizes are the same as in the design files. Gotta :heart: CSS Grids!